scraper/tfs/__init__.py

```diff
@@ -0,0 +1,187 @@
+#! /usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+import logging
+import os
+
+from vsts.vss_connection import VssConnection
+from msrest.authentication import BasicAuthentication
+from scraper.tfs.models import TFSProject
+
+logger = logging.getLogger(__name__)
+
+HARD_CODED_TOP = 10000
+
+
+def get_projects_metadata(baseurl, token):
+    logger.debug('Retrieving TFS Metdata.....')
+    return populate_tfs_projects(baseurl, token)
+
+
+def populate_tfs_projects(baseurl, token):
+    tfs_project_info_collection = []
+    projects = get_all_projects(baseurl, token)
+
+    for project in projects:
```

Comment:
FWIW, this method could be simplified with a list comprehension as: 

```python
def populate_tfs_projects(baseurl, token):
    projects = get_all_projects(baseurl, token)
    return [project for project in get_all_projects(baseurl, token)]
```

Reply:
Agree, sort of. [Method now uses](https://github.com/CDCgov/scraper/commit/982157f3c3bdb2c904478957880b058a57491a20#diff-61198513656b34ac7e662c06d9dba06bR16):
```python
def get_projects_metadata(baseurl, token):
    logger.debug('Retrieving TFS Metdata.....')
    return [project for project in get_all_projects(baseurl, token)]
```